02week/ticTacToe.js

```diff
@@ -24,39 +24,94 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  return board[0].every(square => square === playerTurn) || board[1].every(square => square === playerTurn) || board[2].every(square => square === playerTurn);
 }
 
 function verticalWin() {
-  // Your code here
+  return [board[0][0], board[1][0], board[2][0]].every(square => square === playerTurn) || [board[0][1], board[1][1], board[2][1]].every(square => square === playerTurn) || [board[0][2], board[1][2], board[2][2]].every(square => square === playerTurn);
 }
 
 function diagonalWin() {
-  // Your code here
+  return [board[0][0], board[1][1], board[2][2]].every(square => square === playerTurn) || [board[0][2], board[1][1], board[2][0]].every(square => square === playerTurn);
```

Comment:
Nice use of the `.every() method` here.

![nicely done](http://www.reactiongifs.com/r/2012/08/well_done_sir.gif)